quint/package.json

```diff
@@ -88,7 +88,7 @@
     "yargs": "^17.2.1"
   },
   "scripts": {
-    "compile": "tsc && copyfiles ./src/builtin.qnt ./dist/src/",
+    "compile": "tsc && copyfiles -u 1 ./src/builtin.qnt ./dist/src/",
```

Comment:
what is `copyfiles`? I don't have such a command on MacOS

Reply:
It is a typescript library that handles copying files in multiple OS. `npm install` for `quint` will install it for you.

Reply:
ah, ok, thanks for the clarification!